### PR TITLE
Feature/better oauth mode detection

### PR DESCRIPTION
**Pull request by leomendoza123**

### CODE REVIEW

**Comment by leomendoza123** [comment]
[The oauth session is forgotten](https://trello.com/c/mwId6nLd/7036-the-oauth-session-is-forgotten)

**Comment by leomendoza123** [comment]
[[QA] Sandbox, sign out issue during import wizard OAuth](https://trello.com/c/6JmIIBuW/7026-qa-sandbox-sign-out-issue-during-import-wizard-oauth)

File: src/app/constants.ts [hunk 1]
```diff
@@ -36,6 +36,19 @@ export const ApplicationRoutes = {
   home: '',
 }
 
+export const HeadlessOnOauthRoutes = [
+  ApplicationRoutes.twoFactor,
+  ApplicationRoutes.institutionalLinking,
+  ApplicationRoutes.social,
+  ApplicationRoutes.institutionalLinking,
```
Comment (DanielPalafox) [inline src/app/constants.ts:43]: This one is duplicated

**Review by DanielPalafox — CHANGES_REQUESTED** [review]

**Review by DanielPalafox — APPROVED** [review]

**Comment by leomendoza123** [comment]
[[QA] OAuth is interrupted if you cancel alternate signin option and go back to normal signin](https://trello.com/c/H7Nu4p9T/7021-qa-oauth-is-interrupted-if-you-cancel-alternate-signin-option-and-go-back-to-normal-signin)

**Comment by leomendoza123** [comment]
[A strange scenario where the homepage shows without footer/header](https://trello.com/c/gz8wIPeJ/7035-a-strange-scenario-where-the-homepage-shows-without-footer-header)

**Pull request merged**